db/data/20220425143919_fix_incorrectly_assigned_philosophy_subject_specialisms.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class FixIncorrectlyAssignedPhilosophySubjectSpecialisms < ActiveRecord::Migration[6.1]
+  def up
+    trainees = Trainee.where(course_subject_one: CourseSubjects::PHILOSOPHY).or(Trainee.where(course_subject_two: CourseSubjects::PHILOSOPHY)).or(Trainee.where(course_subject_three: CourseSubjects::PHILOSOPHY)).where(created_from_dttp: true)
+    trainees.each do |trainee|
+      if trainee.dttp_trainee.latest_placement_assignment.response["_dfe_ittsubject1id_value"] == RELIGIOUS_EDUCATION_DTTP_ID
```

Comment:
I don't think this is defined `RELIGIOUS_EDUCATION_DTTP_ID`. As per @edwardhorsford 's comment it should reference the codeset RELIGIOUS_STUDIES entity id shouldn't it?